internal/aws/containerinsight/const.go

```diff
@@ -157,11 +157,12 @@ const (
 	TypeContainerDiskIO    = "ContainerDiskIO"
 	// Special type for pause container
 	// because containerd does not set container name pause container name to POD like docker does.
-	TypeInfraContainer = "InfraContainer"
-	TypeGpuContainer   = "ContainerGPU"
-	TypeGpuPod         = "PodGPU"
-	TypeGpuNode        = "NodeGPU"
-	TypeGpuCluster     = "ClusterGPU"
+	TypeInfraContainer  = "InfraContainer"
+	TypeGpuContainer    = "ContainerGPU"
+	TypeGpuPod          = "PodGPU"
+	TypeGpuNode         = "NodeGPU"
+	TypeGpuCluster      = "ClusterGPU"
+	TypeNeuronContainer = "ContainerNeuron"
```

Comment:
Is this to take a different path from nvidia gpu for decoration?

Reply:
Yes the decoration is although similar to GPU but we want the source labels to be different in this case